docs/niscope/class.rst

```diff
@@ -1497,6 +1520,106 @@ get_equalization_filter_coefficients
                 niscope.Session repeated capabilities container, and calling this method on the result.
 
 
+get_ext_cal_last_date_and_time
+------------------------------
+
+    .. py:currentmodule:: niscope.Session
+
+    .. py:method:: get_ext_cal_last_date_and_time()
+
+            Returns the date and time of the last external calibration performed.
+
+            
+
+            .. note:: The NI 4050 and NI 4060 are not supported.
```

Comment:
These are DMMs, not digitizers.

Reply:
More precisely, those are EOLed DMMs that we should not even talk about in NI-DMM!